website/translated_docs/ar/learn-consensus.md

```diff
@@ -6,41 +6,51 @@ sidebar_label: Polkadot Consensus
+Although simple and effective in coming to a decentralized consensus, proof of work consensus consumes an incredible amount of energy, has no economic or provable finality, and has no effective strategy in resisting cartels.
+
+## Probabilistic vs. provable finality
+
+A pure Nakamoto consensus blockchain which runs PoW is only able to achieve the notion of _probabilistic finality_ and reach _eventual consensus_. Probabilistic finality means that under some assumptions about the network and participants, if we see a few blocks building on a given block, we can estimate the probability that it is final.  Eventual consensus means that at some point in the future, all nodes will agree on the truthfulness of one set of data.  This eventual consensus may take a long time and will not be able to be determined how long it will take ahead of time.  However, finality gadgets such as GRANDPA or Ethereum's Casper FFG are designed to give stronger and quicker guarantees on the finality of blocks - specifically, that they can never be reverted after some process of byzantine agreements has taken place. The notion of irreversible consensus is known as _provable finality._
 
 In the GRANDPA paper, it is phrased in this way:
 
-> We say an oracle A in a protocol is *eventually consistent* if it returns the same value to all participants after some unspecified time.
+> We say an oracle A in a protocol is _eventually consistent_ if it returns the same value to all participants after some unspecified time.
 
 ## What is GRANDPA/BABE?
 
 ### Hybrid Consensus
 
-There are two acronyms we use when we talk about the consensus protocol of Polkadot, GRANDPA and BABE. We talk about both of these acronyms because Polkadot uses what is known as **hybrid consensus**. Hybrid consensus splits up the finality gadget from the block production mechanism. This allows for blocks to be produced more rapidly and the slower finality mechanism to run in a separate process to finalize blocks without risking slower transaction processing.
+There are two acronyms we use when we talk about the consensus protocol of Polkadot, GRANDPA and BABE. We talk about both of these acronyms because Polkadot uses what is known as **hybrid consensus**. Hybrid consensus splits up the finality gadget from the block production mechanism.
```

Comment:
It looks like elsewhere, keywords are being introduced with underscores ( `_phrase_` ) instead of double-asterisks ( `**phrase**` ).  I'm fine either way but we should be consistent.

Reply:
We should indeed have a style guide for this - do we italicize or bolden introduced keywords? What's prevalent right now?

Reply:
I concur

Reply:
Changed in PR.